### PR TITLE
Cato SASE connector

### PR DESCRIPTION
**Pull request by vg-svitla**

### CODE REVIEW

File: CatoNetwork/cato/connector.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+        return await self._push_events(result)
+
+    def run(self) -> None:  # pragma: no cover
+        """Runs Cato Sase."""
+        loop = asyncio.get_event_loop()
+
+        previous_processing_end = None
+        try:
+            while True:
```
Comment (squioc) [inline CatoNetwork/cato/connector.py:135]: I suggest using `self.running` that is False when the connector should end.
```suggestion
            while self.running:
```

**Review by squioc — APPROVED** [review]
:+1:

File: CatoNetwork/cato/connector.py
Comment (squioc) [inline CatoNetwork/cato/connector.py:1]: Could you change the name of this file for something more explict? like `connector_sase_events.py'?

**Review by squioc — CHANGES_REQUESTED** [review]

File: CatoNetwork/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### 2023-06-13 - 1.0.0
```
Comment (squioc) [inline CatoNetwork/CHANGELOG.md:10]: Are you sure about the date?

Comment (vg-svitla) [reply CatoNetwork/CHANGELOG.md]: Fixed thank you!

**Review by Adamowoc — APPROVED** [review]
LGTM

**Pull request merged**